src/operations/security/TypeDefs.js

```diff
@@ -0,0 +1,39 @@
+const TypeDefs = [`
+  #Pagination data structure#
+  type PaginationCerts {
+    page: Int
+    total: Int
+    has_next: Boolean
```

Comment:
Some attrs are in camelCase format and others in snake_case. Can we use only camelCase?

Reply:
Yes, we can only use camelCase. I will change it.